### PR TITLE
Toc dropdowns class component

### PR DESCRIPTION
**Pull request by Helix-Ian**

### CODE REVIEW

File: dashboard-app/src/components/TemplateOnePages/TableOfContents.js [hunk 1]
```diff
@@ -10,9 +10,25 @@ class TableOfContents extends Component {
...
+            currentSelections : []
+        }
         this.pageNumber = 0
     }
 
+    handleSelection(content, depth) {
+        var selectionArray = this.state.currentSelections
+        if (selectionArray[depth] == content) {
+            selectionArray.length = depth
+            this.setState({currentSelections:selectionArray})
```
Comment (david-geisinger) [inline dashboard-app/src/components/TemplateOnePages/TableOfContents.js]: take this and put at bottom

File: dashboard-app/src/components/TemplateOnePages/TableOfContents.js [hunk 2]
```diff
@@ -10,9 +10,25 @@ class TableOfContents extends Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            currentSelections : []
+        }
         this.pageNumber = 0
     }
 
+    handleSelection(content, depth) {
```
Comment (david-geisinger) [inline dashboard-app/src/components/TemplateOnePages/TableOfContents.js:24]: leave a comment for function

Comment (Helix-Ian) [reply dashboard-app/src/components/TemplateOnePages/TableOfContents.js:24]: Resolved

File: dashboard-app/src/components/TemplateOnePages/TableOfContents.js (hunk 1 above)
Comment (Helix-Ian) [reply dashboard-app/src/components/TemplateOnePages/TableOfContents.js]: resolved

**Review by david-geisinger — APPROVED** [review]

**Pull request merged**